packages/kovan-faucets/src/ts/global.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'elliptic';
+declare module 'rollbar';
+declare module 'ethereumjs-tx';
+declare module 'es6-promisify';
+declare module 'web3-provider-engine';
```

Comment:
Copy the type from 0x.js

Reply:
for which one? look like 0x.js doesn't have typings for web3-provider-engine